datasets/cta/exporter.py

```diff
@@ -0,0 +1,182 @@
+try:
```

Comment:
Please always leave at a docstring at the top of every Python file or script.
At least one line saying what it is.

People will copy files and email them around and stuff, so even if there is a README` here, please still put at least one line.